web/modules/custom/dbcdk_community/tests/src/Unit/Content/FlaggableContentRepositoryTest.php

```diff
@@ -145,13 +153,50 @@ public function unreadWithFlagsDataProvider() {
   }
 
   /**
+   * Test that collections can be fetched by the repository.
+   */
+  public function testContentWithCollections() {
+    // Data.
+    $post = new Post();
+    $post->setId(1);
+    $image_collection = new ImageCollection();
+    $image_collection->setId(1);
+    $image_collection->setPostImageCollection($post->getId());
+    $video_collection = new VideoCollection();
+    $video_collection->setId(2);
+    $video_collection->setPostVideoCollection($post->getId());
+
+    // Stubs.
+    $flag_api_stub = $this->getMock('DBCDK\CommunityServices\Api\FlagApi');
+    $post_api_stub = $this->getMock('DBCDK\CommunityServices\Api\PostApi');
+    $comment_api_stub = $this->getMock('DBCDK\CommunityServices\Api\CommentApi');
+    $post_api_stub->method('postPrototypeGetImage')->willReturn([$image_collection]);
+    $post_api_stub->method('postPrototypeGetVideo')->willReturn([$video_collection]);
+
+    // Get a FlaggableContentRepository with a Post object.
+    $flaggable_content = new FlaggableContent($post);
+    $repo = new FlaggableContentRepository($flag_api_stub, $post_api_stub, $comment_api_stub);
+
+    // Assume that the ImageCollection defined above is the same one returned
+    // by the FlaggableContentRepository::getCollections method.
+    $images = $repo->getCollections($flaggable_content, 'image');
+    $this->assertEquals($images[0], $image_collection);
```

Comment:
How about calling assertEquals on the entire `$images` array? This only ensures that the first returned is the expected one - not that any additional collections are not returned.